openverse_catalog/dags/maintenance/pr_review_reminders/pr_review_reminders.py

```diff
@@ -89,20 +94,27 @@ def get_urgency_if_urgent(pr: dict) -> Optional[ReviewDelta]:
 
 
 COMMENT_TEMPLATE = (
-    """
-Based on the {urgency_label} urgency of this PR, the following reviewers are being
+    """\
+Based on the {urgency_label} urgency of this PR, the following reviewers are being \
 gently reminded to review this PR:
 
 {user_logins}
-
 """
     f"{COMMENT_MARKER}"
     """
-Ignoring weekend days, this PR was updated {days_since_update} day(s) ago. PRs
-labelled with {urgency_label} urgency are expected to be reviewed within {urgency_days}.
 
-@{pr_author}, if this PR is not ready for a review, please draft it to prevent reviewers
-from getting further unnecessary pings.
+Ignoring weekend[^1] days, this PR was updated {days_since_update} day(s) ago. \
```

Comment:
```suggestion
Excluding weekend[^1] days, this PR was updated {days_since_update} day(s) ago. \
```
Just a suggestion for clarity.